src/sentry/search/events/builder/profile_functions_metrics.py

```diff
@@ -0,0 +1,50 @@
+from snuba_sdk import Condition, Granularity
+
+from sentry.search.events.builder import (
+    MetricsQueryBuilder,
+    TimeseriesMetricQueryBuilder,
+    TopMetricsQueryBuilder,
+)
+from sentry.search.events.types import SelectType
+
+
+class ProfileFunctionsMetricsQueryBuilder(MetricsQueryBuilder):
+    requires_organization_condition = True
+    profile_functions_metrics_builder = True
+
+    column_remapping = {
+        # We want to remap `message` to `name` for the free
+        # text search use case so that it searches the `name`
+        # (function name) when the user performs a free text search
+        "message": "name",
+    }
+
+    @property
+    def use_default_tags(self) -> bool:
+        return False
```

Comment:
I think you want to use default tags here (I'm going to remove this as a config later) and add that constants list of the tags that exist for the profiling metrics